scene/3d/voxelizer.cpp

```diff
@@ -975,6 +977,9 @@ Ref<MultiMesh> Voxelizer::create_debug_multimesh() {
 	}
 
 	{
+		// This can be a StandardMaterial3D as we already read properties from the BaseMaterial3D.
+		// Albedo and emission textures (the only textures read by the voxelizer) are identical
+		// between StandardMaterial3D and ORMMaterial3D.
```

Comment:
This comment is necessary. The below code creates a new StandardMaterial3D for debug drawing. There is no need for it to be related to the static meshes included in the bake